vms/components/chain/state.go

```diff
@@ -17,8 +17,17 @@ import (
 	"github.com/ava-labs/avalanchego/snow/choices"
 	"github.com/ava-labs/avalanchego/snow/consensus/snowman"
 	"github.com/ava-labs/avalanchego/snow/engine/snowman/block"
+	"github.com/ava-labs/avalanchego/utils/constants"
 )
 
+func cachedBlockSize(_ ids.ID, bw *BlockWrapper) int {
+	return ids.IDLen + len(bw.Bytes()) + 2*constants.PointerOverhead
```

Comment:
Why is this 2x? 1x for inner block pointer and one for the pointer to the wrapper object?

Reply:
> 1x for inner block pointer and one for the pointer to the wrapper object?
Correct - went through and made sure I was consistent (there was 1 case a tx wrapper wasn't being accounted for correctly)

Reply:
May make sense to comment this but that's def a nit